src/mame/bandai/wswan_v.cpp

```diff
@@ -38,9 +49,8 @@ wswan_video_device::~wswan_video_device()
 void wswan_video_device::common_save()
 {
 	save_item(NAME(m_bitmap));
-	save_item(NAME(m_vram));
+	save_pointer(NAME(m_vram), vram_size());
 	save_item(NAME(m_palette_port));
-	save_item(NAME(m_pal));
 	save_item(NAME(m_regs));
 
```

Comment:
Is there a good reason for having `common_save` at all?  Shouldn’t it just be doing this stuff in `device_start`?